relayer/chains/evm/listener.go

```diff
@@ -72,7 +80,11 @@ func (p *Provider) Listener(ctx context.Context, lastSavedHeight uint64, blockIn
 			return nil
 		case <-subscribeStart.C:
 			subscribeStart.Stop()
-			go p.Subscribe(ctx, blockInfoChan)
+			go p.Subscribe(ctx, blockInfoChan, resetFunc)
+		case <-pollHeightTicker.C:
+			pollHeightTicker.Stop()
+			next = p.GetLastSavedBlockHeight()
+			latest = p.latestHeight()
```

Comment:
Why not keep this at the same ticker? Since the are linked to always trigger together currently

Reply:
yeah sure. make sense.

Reply:
nope, we cannot, we have priority set above for the start height and this would mangle the precedence.